doc/src/bond_cpp.md

```diff
@@ -1775,6 +1775,10 @@ to reuse the pre-instantiated code. The one exception are applications using
 custom protocols - by definition templates pre-instantiated at the time Bond
 library was built can't support custom protocols.
 
+If header-only consumption of the library is not required, then defining `BOND_LIB_TYPE`
+macro to `BOND_LIB_TYPE_STATIC` will pre-compile more common code into `bond`
```

Comment:
Minor: "...into the `bond.lib`/`libbond.a` library..."

Reply:
Updated.